src/towncrier/test/test_build.py

```diff
@@ -12,13 +12,14 @@
 from .._shell import cli
 
 
-def setup_simple_project():
+def setup_simple_project(create_newsfragments_directory=True):
     with open("pyproject.toml", "w") as f:
         f.write("[tool.towncrier]\n" 'package = "foo"\n')
     os.mkdir("foo")
     with open("foo/__init__.py", "w") as f:
         f.write('__version__ = "1.2.3"\n')
-    os.mkdir("foo/newsfragments")
+    if create_newsfragments_directory:
```

Comment:
Is it worth updating this setup method when the new functionality is only used by a single test?

Also, the test `test_no_newsfragments` is easier to read when ` os.mkdir("foo/newsfragments")` is called inside the test.